package.json

```diff
@@ -7,7 +7,9 @@
     "node": ">=14"
   },
   "resolutions": {
-    "**/@types/node": "14"
+    "**/@types/node": "14",
+    "@types/react": "18.0.15",
+    "@types/react-dom": "18.0.6"
```

Comment:
I'm not a huge fan of resolutions since it does not solve the same problem for downstream extensions and applications. We should probably think of using [react-window](https://github.com/bvaughn/react-window) instead of `react-virtualized` which is written by the same author and is a replacement for the latter ([related](https://github.com/bvaughn/react-window#how-is-react-window-different-from-react-virtualized)). We can tackle it later if needed but we should probably mention something in the migration guide?

Reply:
I added a note in the migration guide.